document/core/Makefile

```diff
@@ -155,7 +156,7 @@ bikeshed: $(GENERATED)
 	@echo
 	@echo =========================================================================
 	mkdir -p $(BUILDDIR)/bikeshed_mathjax/
-	bikeshed spec --md-status=$(W3C_STATUS) index.bs $(BUILDDIR)/bikeshed_mathjax/index.html
+	bikeshed spec --md-status=$(W3C_STATUS) --md-deadline=$(DEADLINE) index.bs $(BUILDDIR)/bikeshed_mathjax/index.html
```

Comment:
The `Deadline` metadata field could be added to the metadata block at the top of index.bs as well (it would have to be an arbitrary date or `now`); I decided not to do that, with the consequence that the CR/CRD will fail to build if `--md-deadline` is not specified on the command line. This seemed OK, it just means nobody can "accidentally" make a CR. The ED/WD forms are not affected.